src/store/saga.js

```diff
@@ -1,9 +1,12 @@
-import { all, fork } from 'redux-saga/effects';
+import { fork } from 'redux-saga/effects';
 
 import { watcher } from './books/books.sagas';
+import { gaipSaga } from './GAPI/GAPI.sagas';
+import { loginSaga } from './login/login.sagas';
 
 const rootSaga = function* root() {
-    yield all([fork(watcher)]);
+    yield fork(watcher);
+    yield fork(gaipSaga);
+    yield fork(loginSaga);
```

Comment:
1. There's a typo on line 4
2. Is there a reason why all of them are forked one below another ? 
Why not `yield all([fork(watcher), fork(gapiSaga), fork(loginSaga)])`

Reply:
you are 100% right , i totally forgot about it